steelthread-0/walkthrough.md

```diff
@@ -25,62 +28,68 @@ You should also take the opportunity to login your oc CLI as this user. You can
+This is the HTTP endpoint that will revieve the requests from the Fuse Ignite 
+integreation we create and add new fruits to the inventory.
 
-- messagingHost
-- username
-- password
 
-open the Fuse Ignite route and login in to the Fuse Ignite console using the evals@example.com user 
+Load up the launcher application. This is at ```http://launcher-launcher.<BASE_MASTER_DOMAIN> ``` 
 
-- Select connections and create connection
-- Find the AMQP connection and select it. 
-- Fill in the Connection URI Field with the a URI that is in the following format  ```amqp://<messagingHost>:5672?amqp.saslMechanisms=PLAIN```
-- Fill in the username and password fields
-- Disable Check Certificates
-- Click Validate 
-- Click the blue ```next``` button at the top right
-- Add a connection name ```enmaasse```
-- Click Next / Save
+- Click on launch your project
+- Login when prompted using the ```evals@example.com``` user
+- The create application form will show. Choose ```eval``` as the name (this equates to your namespace)
+- When prompted to select target environment choose : ```Code Locally, Build and Deploy```
+- Click the blue down arrows
+- Under mission select ```Crud```
+- Under runtime select ```Node.js```
+- Click the down arrows
+- Under ``` Authorize Git Provider``` you will need to authorize launcher to have access to your github account.
+- Once authorised, under the repository input change the value to ```crud-app```
+- Once the down arrows go blue click them
+- Click setup application.
+- This will create a repo called crud-app in your github account and deploy the CRUD booster to the eval namespace.
+
+You can try out this application by going to the eval namepsace and clicking on the route created for this new application.
 
 
 # Deploy the messaging booster
+**TODO at the moment we do this manually in the future it will be done via launcher**
 
-clone the following booster git@github.com:ssorj/nodejs-messaging-work-queue.git
+Fork the following booster https://github.com/ssorj/nodejs-messaging-work-queue
+into your own repo. We need to fork it as we will be adding some code changes later.
 
-```
-git clone git@github.com:ssorj/nodejs-messaging-work-queue.git
+Next clone the repo to your local machine
 
+```
+git clone git@github.com:<YOUR_GITHUB_HANDLE>/nodejs-messaging-work-queue.gi
 ```
 
 set up the templates in the cluster
 
 ```
 cd nodejs-messaging-work-queue
 oc apply -f templates
-
 ```
-Deploy the applications
+Deploy the frontend application
 
 ```
-oc new-app --template=nodejs-messaging-work-queue-frontend
-oc new-app --template=nodejs-messaging-work-queue-worker
+oc new-app --template=nodejs-messaging-work-queue-frontend -p SOURCE_REPOSITORY_URL=https://github.com/<YOUR_GITHUB_ACCOUNT>/nodejs-messaging-work-queue.git
 ```
 
 Edit the messaging-service configmap
 
 ```
-oc edit cm messaging-service -n eval:
+oc edit cm messaging-service -n eval
 ```
 
 Add in the values from the secret from the Enmasse binding secret
```

Comment:
It's not clear what the Enmasse binding secret is in this case. When viewing the list of secrets in the eval namespace there are numerous Enmasse secrets. We can probably be more specific. Something like ```enmasse-standard-cfj4d-credentials-eikyp```

Reply:
agreed. Will using this terminology  `in the format 'enmasse-standard-xxxxx-credentials-yyyyy'`